types/uc.micro/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "private": true,
+    "name": "@types/uc.micro",
+    "version": "2.1.0.9999",
```

Comment:
the dt-gen command generates versions like this (now?) but on other packages it's similar to `<major>.<minor>.9999`

Reply:
Ah I had reverted that. Looks like new packages cannot be added in the same PR. I'll create a separate one for adding types for uc.micro. But I don't think many people use that package directly.

Reply:
that's the other story I was about to ask you, cool!